packages/fetch/src/request-options.interface.ts

```diff
@@ -2,7 +2,7 @@ import { FetchOptions } from './fetch.interface';
 
 interface ExtraOptions {
   json?: any;
-  headers?: Record<string, string>;
+  headers?: Record<string, string> | Map<string, string>;
```

Comment:
tried to use `Headers` but proved to be more complex than just supporting `Map`
also `undici` uses its own `Headers` which adds some additional complication